cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -106,6 +106,7 @@ def _provisioner(template, provisioning_data, delayed=None):
     return _provisioner
 
 
+@pytest.mark.rhv1
```

Comment:
If we do refer to VM provision as T1:
We had bugs around ram/cpu changes during provision - considering that I thought it would be good to catch those on T1. We can change it to T2, and postpone the discovery of such bugs to the second level. What do you think?

Reply:
Now that I understand the philosophy behing tiers more, I can agree that basic provisioning is T1. This test then perfectly fits into T2 definition I'd say.

Reply:
ok. I'll move it to T2